src/jwks.cc

```diff
@@ -329,8 +295,8 @@ Status extractX509(const std::string& key, Jwks::Pubkey* jwk) {
   if (jwk->x509_ == nullptr) {
     return Status::JwksX509ParseError;
   }
-  jwk->evp_pkey_.reset(X509_get_pubkey(jwk->x509_.get()));
-  if (jwk->evp_pkey_ == nullptr) {
+  jwk->rsa_.reset(EVP_PKEY_get1_RSA(X509_get_pubkey(jwk->x509_.get())));
```

Comment:
does EVP_PKEY_get1_RSA() handle nullptr input?  X509_get_pubkey may return nullptr.

Reply:
It does not, good catch.